chia/data_layer/data_layer_server.py

```diff
@@ -58,3 +102,41 @@ async def file_handler(self, request: web.Request) -> web.Response:
             body=content,
         )
         return response
+
+    def _accept_signal(self, signal_number: int, stack_frame: Any = None) -> None:
+        self.log.info("Got SIGINT or SIGTERM signal - stopping")
+
+        self.stop()
+
+
+async def async_start(config: Optional[str]) -> int:
+
+    shutdown_event = asyncio.Event()
+
+    if config:
+        root_path = Path(config)
+    else:
+        root_path = DEFAULT_ROOT_PATH
+
+    dl_config = load_config(root_path=root_path, filename="config.yaml", sub_config=SERVICE_NAME)
+    setproctitle("data_layer_http")
+    initialize_logging(
+        service_name="data_layer_http",
+        logging_config=dl_config["logging"],
+        root_path=root_path,
+    )
+
+    data_layer_server = DataLayerServer(root_path, dl_config, log, shutdown_event)
+    await data_layer_server.start()
+
+    return 0
+
+
+@click.command()
+@click.option("-c", "--config", type=click.Path(exists=True, writable=True, dir_okay=True), default=None)
```

Comment:
I think calling it `root_path` instead of `config` would be more suggestive of what this represents.

Reply:
updated to root-path and other changes